modules/mcuboot/CMakeLists.txt

```diff
@@ -183,6 +183,13 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       mcuboot
       ${ZEPHYR_NRF_MODULE_DIR}/subsys/partition_manager/ext_flash_mcuboot_secondary.conf
       )
+    if (CONFIG_SPI_NOR)
+    # Overriding the default 'n' of MULTITHREADING in MCUBoot child image
+    add_overlay_config(
+      mcuboot
+      ${ZEPHYR_NRF_MODULE_DIR}/subsys/partition_manager/multithreading.conf
```

Comment:
I know tis is a bit weird but I just couldn't get past https://github.com/nrfconnect/sdk-mcuboot/blob/800d4756702f07364f32538a6f0263c16be3eafc/boot/zephyr/Kconfig#L682 overriding my symbol without doing this.

Another option is that I open another PR to the MCUBoot repository with a change to the Kconfig `&& !SPI_NOR` for the `MULTITHREADING`.

Reply:
Where did you put your override? In `modules/mcuboot/boot/zephyr/Kconfig` ?

Reply:
I `CONFIG_SPI_NOR` also enabled inside the child image ?

Should we propagate the SPI_NOR setting from parent to child ?

Reply:
This doesn't work as expected.

If I try to build:
```
$ cmake -GNinja -Bbuild -DBOARD=nrf9160dk_nrf9160_ns@0.14.0 samples/nrf9160/http_update/full_modem_update/ -DCONFIG_BOOTLOADER_MCUBOOT=y
....
=== child image mcuboot -  begin ===
Including boilerplate (Zephyr workspace): /projects/github/ncs/zephyr/cmake/app/boilerplate.cmake
-- Application: /projects/github/ncs/bootloader/mcuboot/boot/zephyr
...
Loaded configuration '/projects/github/ncs/zephyr/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_defconfig'
Merged configuration '/projects/github/ncs/bootloader/mcuboot/boot/zephyr/prj.conf'
Merged configuration '/projects/github/ncs/nrf/subsys/partition_manager/partition_manager_enabled.conf'
...
=== child image mcuboot -  end ===
...
```

doesn't enable multithreading:
```
$ grep CONFIG_MULTITHREADING build/mcuboot/zephyr/.config 
# CONFIG_MULTITHREADING is not set
```

So something is not working, even with this PR.

Reply:
Hmm that's weird I'll look into it.

Reply:
let me know when you find the reason.